src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Buergerhaushalt/Context/Module.ts

```diff
@@ -1,32 +1,16 @@
 import * as AdhEmbedModule from "../../../Embed/Module";
-import * as AdhResourceAreaModule from "../../../ResourceArea/Module";
-
-import * as AdhMeinberlinBuergerhaushaltWorkbenchModule from "../Workbench/Module";
 
 import * as AdhEmbed from "../../../Embed/Embed";
-import * as AdhResourceArea from "../../../ResourceArea/ResourceArea";
-
-import * as AdhMeinberlinBuergerhaushaltWorkbench from "../Workbench/Workbench";
-
-import RIBuergerhaushaltProcess from "../../../../Resources_/adhocracy_meinberlin/resources/burgerhaushalt/IProcess";
 
 
 export var moduleName = "adhMeinberlinBuergerhaushaltContext";
 
 export var register = (angular) => {
     angular
         .module(moduleName, [
-            AdhEmbedModule.moduleName,
-            AdhMeinberlinBuergerhaushaltWorkbenchModule.moduleName,
-            AdhResourceAreaModule.moduleName
+            AdhEmbedModule.moduleName
         ])
-        .config(["adhEmbedProvider", (adhEmbedProvider : AdhEmbed.Provider) => {
+        .config(["adhEmbedProvider", (adhEmbedProvider: AdhEmbed.Provider) => {
             adhEmbedProvider.registerContext("buergerhaushalt", ["burgerhaushalt"]);
-        }])
-        .config(["adhResourceAreaProvider", (adhResourceAreaProvider : AdhResourceArea.Provider) => {
-            AdhMeinberlinBuergerhaushaltWorkbench.registerRoutes(
-                RIBuergerhaushaltProcess.content_type,
-                "buergerhaushalt"
-            )(adhResourceAreaProvider);
```

Comment:
I think this is still needed, except with AdhMeinberlinIdeaCollection istead of AdhMeinberlinBuergerhaushaltWorkbench.